.github/workflows/regression_tests.yml

```diff
@@ -64,11 +75,25 @@ jobs:
           source loki-activate
           pip install cmake
 
-      - name: Run CLOUDSC regression tests
+      - name: Install ECWAM dependencies
+        run: |
+          sudo apt-get -o Acquire::Retries=3 install -y libopenmpi-dev
+          source loki-activate
+          pip install fypp
```

Comment:
[No action required] This really ought to be happening in the FIELD_API CMake (I'll get to that, promised!)

Reply:
So FIELD_API should install fypp if not found? I also use fypp in ecWam: I use a yaml config file to define the derived-types and use fypp to generate the FIELD_API glue code. So fypp and pyyaml need to be installed on the same interpreter. That's why I source loki and then install fypp, since the ecWam build is done with `loki-env` activated and it already has pyyaml.

Reply:
Yeah, the idea would essentially be to export `fypp` as a CMake target. If it doesn't exist, yet, try a `find_program` and export it, otherwise create a venv and install it in there. That pattern can be used in all projects that use fypp and the first in any bundle will automatically become the one to install it. But I'll do this soon ™️